tests/nnfw_api/src/one_op_tests/Add.cc

```diff
@@ -84,6 +84,25 @@ TEST_F(GenModelTest, OneOp_Add_VarToVarSame)
   SUCCEED();
 }
 
+TEST_F(GenModelTest, OneOp_Add_VarToVarSize0)
+{
+  CircleGen cgen;
+  int a = cgen.addTensor({{0}, circle::TensorType::TensorType_FLOAT32});
+  int b = cgen.addTensor({{0}, circle::TensorType::TensorType_FLOAT32});
+  int c = cgen.addTensor({{0}, circle::TensorType::TensorType_FLOAT32});
+  int m = cgen.addTensor({{0}, circle::TensorType::TensorType_FLOAT32});
+  int out = cgen.addTensor({{0}, circle::TensorType::TensorType_FLOAT32});
+  cgen.addOperatorAdd({{a, b}, {m}}, circle::ActivationFunctionType_NONE);
+  cgen.addOperatorAdd({{m, c}, {out}}, circle::ActivationFunctionType_NONE);
+  cgen.setInputsAndOutputs({a, b, c}, {out});
+
+  _context = std::make_unique<GenModelTestContext>(cgen.finish());
+  _context->addTestCase(uniformTCD<float>({{}, {}, {}}, {{}}));
```

Comment:
Shouln't this be the following?
```suggestion
  _context->addTestCase(uniformTCD<float>({{0}, {0}, {0}}, {{0}}));
```

Reply:
Since the shape is `{0}`, the buffer must be empty. Please note that the former is "shape" and the latter is "data".